### PR TITLE
fixed .lib paths in nuget packaging

### LINKED ISSUE #1583: librdkafka.redist.targets in the v0.11.3 nuget package has invalid path
Description
===========
When building on MSBuild we use the librdkafka.redist.targets in our project files.  However the librdkafka.lib is being referenced as $(MSBuildThisFileDirectory)..\..\runtimes\win7-x64\native\librdkafka.lib but the path should be $(MSBuildThisFileDirectory)..\..\build\native\lib\win7\x64\win7-x64-Release\v120\librdkafka.lib

How to reproduce
================
Try to include the v0.11.3 nuget package and include the librdkafka.redist.targets file to link against librdkafka.lib.  The path is not valid and will not be able to link using this targets file.

Checklist
=========

**IMPORTANT**: We will close issues where the checklist has not been completed.

Please provide the following information:

 - [x] librdkafka version (v0.11.3):
 - [ ] Apache Kafka version: `e.g., 0.10.2.3`
 - [ ] librdkafka client configuration: `e.g., message.timeout.ms=123, auto.reset.offset=earliest, ..`
 - [ ] Operating system: `e.g., Centos 5 (x64)`
 - [ ] Provide logs (with `debug=..` as necessary) from librdkafka
 - [ ] Provide broker log excerpts
 - [ ] Critical issue

### PR DESCRIPTION
**Pull request by mhowlett**
@edenhill - fixes #1583

### CODE REVIEW

**Comment by mhowlett** [comment]
tested by building an x64 c++ project that references this.

File: packaging/nuget/templates/librdkafka.redist.targets [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemDefinitionGroup>
     <Link>
-      <AdditionalDependencies Condition="'$(Platform)' == 'x64'">$(MSBuildThisFileDirectory)..\..\runtimes\win7-x64\native\librdkafka.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <AdditionalDependencies Condition="'$(Platform)' != 'x64'">$(MSBuildThisFileDirectory)..\..\runtimes\win7-x86\native\librdkafka.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <AdditionalLibraryDirectories Condition="'$(Platform)' == 'x64'">$(MSBuildThisFileDirectory)..\..\runtimes\win7-x64\native;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
-      <AdditionalLibraryDirectories Condition="'$(Platform)' != 'x64'">$(MSBuildThisFileDirectory)..\..\runtimes\win7-x86\native;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+      <AdditionalDependencies Condition="'$(Platform)' == 'x64'">$(MSBuildThisFileDirectory)..\..\build\native\lib\win7\x64\win7-x64-Release\v120\librdkafka.lib;%(AdditionalDependencies)</AdditionalDependencies>
```
Comment (edenhill) [inline packaging/nuget/templates/librdkafka.redist.targets]: Unnecessary dir-backtracking, the lib folder is in the same directory as the target sfile:
```
     1459  2017-11-30 07:49   build/native/librdkafka.redist.targets
   125600  2017-11-30 07:49   build/native/include/librdkafka/rdkafka.h
    70853  2017-11-30 07:49   build/native/include/librdkafka/rdkafkacpp.h
    39848  2017-11-30 07:49   build/native/lib/win7/x64/win7-x64-Release/v120/librdkafka.lib
    39098  2017-11-30 07:49   build/native/lib/win7/x64/win7-x64-Release/v120/librdkafkacpp.lib
    40610  2017-11-30 07:49   build/native/lib/win7/x86/win7-x86-Release/v120/librdkafka.lib
    38312  2017-11-30 07:49   build/native/lib/win7/x86/win7-x86-Release/v120/librdkafkacpp.lib
```

File: packaging/nuget/templates/librdkafka.redist.targets [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+      <AdditionalDependencies Condition="'$(Platform)' != 'x64'">$(MSBuildThisFileDirectory)..\..\build\native\lib\win7\x86\win7-x86-Release\v120\librdkafka.lib;%(AdditionalDependencies)</AdditionalDependencies>
```
Comment (edenhill) [inline packaging/nuget/templates/librdkafka.redist.targets]: librdkafkacpp.lib x 2 is missing

**Comment by edenhill** [comment]
> tested by building an x64 c++ project that references this.

that was referencing the C library though, right?

File: packaging/nuget/templates/librdkafka.redist.targets (hunk 2 above)
Comment (mhowlett) [reply packaging/nuget/templates/librdkafka.redist.targets]: would that mean the cpp library is linked with C programs that reference the nuget package as well? (not clear on the details). do we really want this?

we have never referenced the cpp .libs in this file btw.

File: packaging/nuget/templates/librdkafka.redist.targets (hunk 1 above)
Comment (mhowlett) [reply packaging/nuget/templates/librdkafka.redist.targets]: good point

**Comment by mhowlett** [comment]
re-checked build of an empty C++ project that references the new nuget package after path updates suggested in review. the build completes. In 0.11.3, the build fails with a linker error (cannot find file).

**Pull request merged**